test/integration/scenarios/04.workflow/manifest/campaign.yaml

```diff
@@ -35,6 +35,7 @@ spec:
       name: deploy
       provider: providers.stage.materialize
       stageSelector: ""
+      schedule: "2020-10-31T12:00:00-07:00"
```

Comment:
@RemindD  will this affect the test? This is a timestamp in the past.

Reply:
It's OK. It will still persist a schedule event in the state store and job manager will immediately trigger the stage once it's found.